README.md

```diff
@@ -1 +1,66 @@
-# stw-stubs
+# SWT Stub API
```

Comment:
STW Stub API

Reply:
I think this might have come from an older commit but has since been corrected. Might have been showing as there was merge conflicts on the PR.